code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -301,9 +341,20 @@ var/static/regex/multispin_words = regex("like a record baby")
 
 	//MOVE
 	else if((findtext(message, move_words)))
-		for(var/V in listeners)
-			var/mob/living/L = V
-			step(L, pick(cardinal))
+		var/direction
+		if(findtext(message, up_words))
+			direction = NORTH
+		else if(findtext(message, down_words))
+			direction = SOUTH
+		else if(findtext(message, left_words))
+			direction = WEST
+		else if(findtext(message, right_words))
+			direction = EAST
+		for(var/i=1, i<=(5*power_multiplier), i++)
+			for(var/V in listeners)
+				var/mob/living/L = V
+				step(L, direction ? direction : pick(cardinal))
+			sleep(10)
```

Comment:
this delay means you could do another command before it completes, because it'll do all the sleeps before it sets the cooldown.